profiles/killbill/src/main/java/org/killbill/billing/server/log/obfuscators/ObfuscatorConverter.java

```diff
@@ -61,4 +70,25 @@ public String convert(final ILoggingEvent event) {
         }
         return convertedMessage;
     }
+
+    private void mergeAdditionalObfuscators(final List<String> additionalObfuscators) {
+        if(additionalObfuscators != null) {
+            for (String obfuscatorClassName : additionalObfuscators) {
+                createAndAddObfuscator(obfuscatorClassName);
+            }
+        }
+    }
+
+    private void createAndAddObfuscator(final String obfuscatorClassName) {
+        try {
+            Obfuscator newObfuscator = createDynamicObfuscator(obfuscatorClassName);
+            obfuscators.add(newObfuscator);
+        } catch (Exception e) {
+            // Ignore? Not sure the impact of importing a logger here
+        }
+    }
+
+    private Obfuscator createDynamicObfuscator(final String className) throws Exception {
+        return (Obfuscator) Class.forName(className).getConstructor().newInstance();
```

Comment:
Is it actually working? I wouldn't be surprised if you run into OSGI import/export problems (`ClassCastException`).

One work-around would be to share the `Obfuscator` interface (maybe in https://github.com/killbill/killbill-platform/tree/master/osgi-bundles/libs/killbill/src/main/java/org/killbill/billing/osgi/libs/killbill, package that is already imported both by plugins and by killbill server).